### DISCUSSION TITLE
Follow up on promise to create Connector Ecosystem

### DISCUSSION THREAD

**Discussion by hinogi** [prisma/prisma#15714] [Q&A]
> not only us should be able to write a connector but anyone should be able to write a code to implement a connector
-- Marcus  Böhm

It would really help a lot if that would become reality especially because a lot of people want to do it but cannot and prisma is also not helping in providing connectors that are only for a small group of people like https://github.com/prisma/prisma/issues/2853#issuecomment-1200264294 the connector for oracle db. @mavilein 

https://youtu.be/RAoXdyI_PH4?t=911

**Comment by nikolasburk** [comment]
Hey @hinogi, thanks a lot for bringing this up! At the moment, our Product team has set a clear direction that we're neither going to work on new connectors nor on the connector infrastructure since we first want to make sure that all existing connectors are properly built out. That being said, we still very much want to improve the connector implementations, making it easier for ourselves to build new ones and ultimately enabling other developers to build their own connectors as well 🙏 (I also briefly talked about this at the beginning of my [Prisma Day talk](https://www.youtube.com/watch?v=noKmJyHYZd4&list=PLn2e1F9Rfr6lj7O6z56_lk-GA7RegOFWv&ab_channel=Prisma)).

**Reply by hinogi** [reply]
Just to make sure, you realize that the youtube video is from 2019, so it is more than 3 years now.

**Reply by hinogi** [reply]
It would help to enable the community to contribute connectors.

**Reply by hinogi** [reply]
This delaying tactics will just frustrate ppl and the will tend to something else that supports their infrastructure.

**Reply by charliepan** [reply]
> This delaying tactics will just frustrate ppl and the will tend to something else that supports their infrastructure.

Yup just had to do exactly what you described.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
